src/commands/cloud-run/flare.ts

```diff
@@ -501,3 +537,133 @@ export const saveLogsFile = (logs: CloudRunLog[], filePath: string) => {
   const data = rows.join('\n')
   writeFile(filePath, data)
 }
+
+/**
+ * Gets recent revisions for a cloud-run service
+ * @param service
+ * @param location
+ * @param project
+ * @returns a string array of recent revisions and their deployment timestamp
+ */
+export const getRecentRevisions = async (service: string, location: string, project: string) => {
+  const client = new RevisionsClient()
+  const request = {
+    parent: client.servicePath(project, location, service),
+  }
+  const revisions = (await client.listRevisions(request))[0]
```

Comment:
Previously you would do 

```
(await client.listRevisions(request)) as any
```
But now it's the first element? I'm confused what's happening here.

Reply:
Is it guaranteed that the first element will always exist?

Reply:
Yes, the `listRevisions()` method returns a tuple:
```
Promise<[
        protos.google.cloud.run.v2.IRevision[],
        protos.google.cloud.run.v2.IListRevisionsRequest | null,
        protos.google.cloud.run.v2.IListRevisionsResponse
    ]
```

I am using `[0]` to get the first element, the array of revisions

Reply:
I will add a comment explaining this ^^